plugins/arDominionB5Plugin/modules/search/templates/_box.php

```diff
@@ -7,70 +62,6 @@ class="d-flex flex-grow-1 my-2"
   <input type="hidden" name="topLod" value="0">
   <input type="hidden" name="sort" value="relevance">
   <div class="input-group flex-nowrap">
-    <button
-      id="search-box-options"
-      class="btn btn-sm atom-btn-secondary dropdown-toggle"
-      type="button"
-      data-bs-toggle="dropdown"
-      data-bs-auto-close="outside"
-      aria-expanded="false">
-      <i class="fas fa-cog" aria-hidden="true"></i>
-      <span class="visually-hidden"><?php echo __('Search options'); ?></span>
-    </button>
-    <div class="dropdown-menu mt-2" aria-labelledby="search-box-options">
-      <?php if (sfConfig::get('app_multi_repository')) { ?>
-        <div class="px-3 py-2">
-          <div class="form-check">
-            <input
-              class="form-check-input"
-              type="radio"
-              name="repos"
-              id="search-realm-global"
-              <?php echo !isset($repository) ? 'checked' : ''; ?>
-              value>
-            <label class="form-check-label" for="search-realm-global">
-              <?php echo __('Global search'); ?>
-            </label>
-          </div>
-          <?php if (isset($repository)) { ?>
-            <div class="form-check">
-              <input
-                class="form-check-input"
-                type="radio"
-                name="repos"
-                id="search-realm-repo"
-                checked
-                value="<?php echo $repository->id; ?>">
-              <label class="form-check-label" for="search-realm-repo">
-                <?php echo __('Search <span>%1%</span>', ['%1%' => render_title($repository)]); ?>
-              </label>
-            </div>
-          <?php } ?>
-          <?php if (isset($altRepository)) { ?>
-            <div class="form-check">
-              <input
-                class="form-check-input"
-                type="radio"
-                name="repos"
-                id="search-realm-alt-repo"
-                value="<?php echo $altRepository->id; ?>">
-              <label class="form-check-label" for="search-realm-alt-repo">
-                <?php echo __('Search <span>%1%</span>', ['%1%' => render_title($altRepository)]); ?>
-              </label>
-            </div>
-          <?php } ?>
-        </div>
-        <div class="dropdown-divider"></div>
-      <?php } ?>
-      <a class="dropdown-item" href="<?php echo url_for([
-          'module' => 'informationobject',
-          'action' => 'browse',
-          'showAdvanced' => true,
-          'topLod' => false,
-      ]); ?>">
-        <?php echo __('Advanced search'); ?>
-      </a>
-    </div>
```

Comment:
Moving this drop-down outside the search-box input was done for accessibility reasons. We should test these changes in that regard.

Reply:
Reverting this change due to accessibility concerns, and sticking with just making global search the default option.